resource-aggregate/cqrs/eventstore/mongodb/eventstore.go

```diff
@@ -26,20 +24,40 @@ import (
 
 const eventCName = "events"
 
-const aggregateIDKey = "aggregateid"
-const aggregateIDStrKey = "aggregateidstr"
-const idKey = "_id"
+// Event
 const versionKey = "version"
 const dataKey = "data"
 const eventTypeKey = "eventtype"
+const isSnapshotKey = "issnapshot"
```

Comment:
What is the benefit of storing it in the database and not deciding in the code in one simple function based on the eventType?

Reply:
?? it's keys of map(document)

Reply:
If the event was a snapshot was previously identified by checking the type of event. Now you store this value in the database. Why?